### ISSUE TITLE
[Azure PowerShell Module] Record

### ISSUE THREAD

**Issue by LucasYao93** [FL-LearningGroup/LucasNote#35]

**Comment by LucasYao93** [comment]
# Share module package with other people
```txt
Hi All,
 
The Az.MachineLearningService engineering package is ready.  You can use Az.Tools.Installer module to install and test it: (Run PowerShell as an administrator)
	1. Install the Az.Tools.Installer module itself using the cmdlet.
			a. Install-Module -Name Az.Tools.Installer
	2. Install the Az.Accounts module using the cmdlet to provide account authentication support for future use.
			a. Install-Module Az.Accounts
	3. Optional. If you have Az.MachineLearningServices installed locally, please uninstall it.
	4. Use the cmdlet of the Az.Tools.Installer module to install the Az.MachineLearningServices we need to use.
			a. Install-AzModule -Path  https://azposhpreview.blob.core.windows.net/public/Az.MachineLearningServices.0.1.0.nupkg
	5. Optional. If the Az.Module installation is successful and does not work, then we need to give additional PowerShell permissions to execute external scripts:
			a. Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope Process
 
Thanks,
Lucas
```

**Comment by LucasYao93** [comment]
# Autorest

# new autorest.powershell version
Hi All,

We have just released 4.0.0-dev.5. And below are the new features included in this build.

[PS] Autorest - Support progress bar for LRO cmdlet
[PS] Autorest - Support external links

To use this version, you may add --use:@autorest/powershell@4.0.0-dev.5 when executing autorest.

Thanks,
Xiaogang
